lib/graphql/card.ts

```diff
@@ -94,8 +106,19 @@ export class Card extends Model<CardModel> {
    * @param args  query parameters
    * @returns     details of the card specified in query parameters
    */
-  async get(args: CardOptions): Promise<CardModel | null> {
+  async get(args: GetCardOptions): Promise<CardModel | null> {
     const result: Query = await this.client.rawQuery(GET_CARD, args);
     return result.viewer.mainAccount?.card ?? null;
   }
+
+  /**
+   * Activates a card
+   *
+   * @param args  query parameters
+   * @returns     activated card
+   */
+  async activate(args: ActivateCardOptions): Promise<CardModel> {
```

Comment:
should I do anything to handle errors here? or should we leave that to the consumer / end user?

Reply:
let's leave it for end-user, at least for now